fix dynamic variables (#32)
requestly/requestly-docs: docs/common/features/_import-settings-from-charles.mdx

If you have been using `Charles Proxy` for network debugging, you may have created a set of settings using the Charles Proxy tools
to modify HTTP requests. You can import these tool settings in Requestly in just few clicks.

The following list contains the tools that you can import from Charles Proxy:

- Block Cookies
- Block lists
- No Caching
- Map Remote
- Map Local
- Rewrite

## Steps to import

:::info

> Due to some technical constraints, Importing **individual settings** from a tool in supported yet.

Reviewer: ```suggestion
Due to some technical constraints, importing **individual settings** is not supported yet.
```

Also we should give the work around for this, ie just select the rule which you want to import from `Tools > Import/Export Settings`
